$NetBSD: patch-aa,v 1.2 2010/09/10 10:50:27 taca Exp $

* Use vendor directory.
* $XLDFLAGS might be nil, so care for it.

--- configure.rb.orig	2008-11-08 04:15:36.000000000 +0000
+++ configure.rb
@@ -1,4 +1,4 @@
-#!/usr/bin/env ruby
+#!/usr/pkg/bin/ruby
 
 # Generated automatically using autoconf.rb version 0.2.4
 
@@ -222,6 +222,7 @@ $bindir = CONFIG["bindir"].sub(prefix, "
 $datadir = CONFIG["datadir"].sub(prefix, "$(prefix)").sub(drive, '')
 $libdir = CONFIG["libdir"].sub(prefix, "$(exec_prefix)").sub(drive, '')
 $archdir = $archdir.sub(prefix, "$(prefix)").sub(drive, '')
+$sitedir = $vendordir
 $sitelibdir = $sitelibdir.sub(prefix, "$(prefix)").sub(drive, '')
 $sitearchdir = $sitearchdir.sub(prefix, "$(prefix)").sub(drive, '')
 $includedir = CONFIG["includedir"].sub(prefix, "$(prefix)").sub(drive, '')
@@ -329,11 +330,15 @@ if $LDFLAGS.to_s.empty? && /mswin32/ =~ 
 end
 $LIBS = CONFIG["LIBS"]
 $XLDFLAGS = CONFIG["XLDFLAGS"].to_s
-$XLDFLAGS.gsub!(/-L\./, "")
-if /mswin32/ !~ RUBY_PLATFORM
-  $XLDFLAGS += " -L$(libdir)"
-elsif RUBY_VERSION >= "1.8"
-  $XLDFLAGS += " #{CONFIG['LIBPATHFLAG'] % '$(libdir)'}"
+if $XLDFLAGS.nil?
+  $XLDFLAGS = ''
+else
+  $XLDFLAGS.gsub!(/-L\./, "")
+  if /mswin32/ !~ RUBY_PLATFORM
+    $XLDFLAGS += " -L$(libdir)"
+  elsif RUBY_VERSION >= "1.8"
+    $XLDFLAGS += " #{CONFIG['LIBPATHFLAG'] % '$(libdir)'}"
+  end
 end
 $DLDFLAGS = CONFIG["DLDFLAGS"]
 $LDSHARED = CONFIG["LDSHARED"]
